locationsharinglib/locationsharinglib.py

```diff
@@ -226,6 +226,27 @@ def get_person_by_nickname(self, nickname):
         return next((person for person in self.get_all_people()
                      if person.nickname.lower() == nickname.lower()), None)
 
+    def get_latitude_by_nickname(self, nickname):
+        """Retrieves a person's latitude by nickname."""
+        person = self.get_person_by_nickname(nickname)
+        if not person:
+            return '', ''
```

Comment:
This should only return one value on person retrieval failure since this method only returns one value on success.